self-core-impl/src/test/resources/replies/commands.comment_en.md

```diff
@@ -0,0 +1,5 @@
+* ``Hello`` -- any comment addressed to the PM containing the word **"hello"**.
+* ``Status`` -- any comment addressed to the PM, containing the word **"status"**. The PM will respond with the Task's status, assignee, deadline etc.
+* ``Resign`` -- a task's assignee can always resign from the task, if they cannot or simply do not want to solve it. The comment should be addressed to the PM and contain the word **"resign"**. Only the task's assignee can give this command.
+* ``Deregister`` -- a task can be deregistered (taken out of scope) with a comment containing the word **"deregister"**. Only a Contributor with role ``PO`` or ``ARCH`` can give this command. When a task is deregistered, the task's assignee is automatically resigned.
+* ``Register`` -- an Issue or PR can be registered as a Task with a comment containing the word **"register"**. Any Contributor can give this command.
```

Comment:
@criske Can you also start this message with: ``Hi @%s! Here are the commands which I understand: ``, where %s should be replaced with the author's username. 

Also, at Register and Deegister, we should specify ``add`` and ``remove`` too. If I remember well, also the Resign command has more options.

If you look into ``commands_en.properties``, some commands have more than one entry, which means the PM will respond with that command for all those entries.